README.md

```diff
@@ -55,7 +55,17 @@ unless real values don't help users know what to change.
 -->
 
 ```hcl
-
+module "db2" {
+  source                      = "terraform-ibm-modules/db2-cloud/ibm"
+  version                     = "1.0.0"
```

Comment:
This is not how we usually put the version in the usage. We usually do:
`version = "X.Y.Z" # Replace "X.Y.Z" with a release version to lock into a specific release`